rest/client/client.go

```diff
@@ -51,27 +51,17 @@ type TaskData struct {
 // the API server. To change the default retry behavior, use the SetTimeoutStart, SetTimeoutMax,
 // and SetMaxAttempts methods.
 func NewCommunicator(serverURL string) Communicator {
-	evergreen := &communicatorImpl{
+	return &communicatorImpl{
 		maxAttempts:  defaultMaxAttempts,
 		timeoutStart: defaultTimeoutStart,
 		timeoutMax:   defaultTimeoutMax,
 		serverURL:    serverURL,
+		httpClient:   util.GetHttpClient(),
 	}
-	evergreen.httpClient = &http.Client{
-		Transport: &http.Transport{
-			Proxy:             http.ProxyFromEnvironment,
-			DisableKeepAlives: true,
-			Dial: (&net.Dialer{
-				Timeout:   30 * time.Second,
-				KeepAlive: 30 * time.Second,
-			}).Dial,
-			TLSHandshakeTimeout: 10 * time.Second,
-		},
-		Timeout: heartbeatTimeout,
```

Comment:
Does the pooled client have a different `Timeout`?

Reply:
fixed